bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ResourceListItemImpl.java

```diff
@@ -95,18 +95,4 @@ public String getPath() {
     public String getName() {
         return name;
     }
-
```

Comment:
Shouldn't this class have a `getComponentData()` method like other components?

Reply:
All of the list item implementations share the same `getComponentData()` method (defined in `AbstractListItemImpl`)
The exceptions are the `LanguageNavigationItemImpl` and `PageListItemImpl` which return PageData.

The `getComponentData()` method could be removed from `AbstractListItem` and pushed down to each of the implementations, but with the exception of those two, it would just be duplication.

Reply:
ok, thx for clarifying. It's fine if this class inherits the `getComponentData()` method from `AbstractListItemImpl`.